### PR TITLE
Swedish translation

### PR DESCRIPTION
**Pull request by MalinAhlberg**
Translations of `app/locales/en.json`, also adds the field `Sweden` to the english translations.

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
   "Everyone who has submitted a report, also has:": "Everyone who has submitted a report, also has:",
   "Total number of reports": "Total number of reports",
   "Positive test results and close contact": "Positive test results and close contact",
-  "Clarifications data basis": "Clarifications data basis",
+  "Clarifications data basis": "Förtydliganden angående databasen",
```
Comment (ewels) [inline app/locales/en.json]: Is this translation intended to be in this file?

File: app/locales/sv.json [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  "Cigarette smoking habits:": "Rökvanor",
+  "I currently smoke": "Jag är rökare",
+  "I used to smoke": "Jag har tidigare varit rökare",
+  "I have never smoked": "Jag har aldrig varit rökare",
+  "What is your current body temperature? (Celcius)": "Vad är din kroppstemperatur, i grader Celsius?",
+  "This field is optional": "Detta fält är frivilligt",
+  "Close": "Stäng",
+  "Contribute now": "Bidra nu",
+  "Help us uncover and predict the corona spread in your area":
+  "Hjälp oss <span>kartlägga</span> och <span>förutse</span> spridningen av corona i ditt område"
```
Comment (mihai-sysbio) [inline app/locales/sv.json]: Is the `<span>` really meant to be used here?

File: app/locales/en.json (hunk 1 above)
Comment (mihai-sysbio) [reply app/locales/en.json]: I wouldn't think so either, since it's `en.json`.

**Comment by mihai-sysbio** [comment]
There's a typo in the project, `Celcius` should be [Celsius](https://en.wikipedia.org/wiki/Celsius). Would it be okay to fix in on this branch, or should it be separate?

**Comment by ewels** [comment]
A mini-PR doesn't hurt, I'm sure that it'll be merged super fast.. If you don't mind doing it :)

**Comment by talavis** [comment]
I think they have decided that the language code is se instead of sv.

**Comment by talavis** [comment]
Must add `se-word-list.txt`.
`app/domain/urls.ts` must have se added; my suggestion:

```diff
diff --git a/app/domain/urls.ts b/app/domain/urls.ts
index f8f2218..6934cbd 100644
--- a/app/domain/urls.ts
+++ b/app/domain/urls.ts
@@ -1,6 +1,6 @@
 import { LANGUAGE } from '../../config.json';
 
-export type Locale = 'no' | 'nl' | 'en' | 'sk' | 'it';
+export type Locale = 'no' | 'nl' | 'en' | 'sk' | 'it' | 'se';
 
 type Urls = {
   [locale in Locale]: {
@@ -45,6 +45,16 @@ const localeAwareUrls: Urls = {
     api: '/api',
     apiDocs: '/api-docs',
     statistics: '/statistics'
+  },
+    se: {
+    submitReport: '/',
+    profile: '/halsotillstand',
+    privacyPolicy: '/integritetspolicy',
+    map: '/karta',
+    contributors: '/frivilliga',
+    api: '/api',
+    apiDocs: '/api-docs',
+    statistics: '/statistik'
   },
   sk: {
     submitReport: '/',
```

File: app/locales/sv.json (hunk 2 above)
Comment (talavis) [reply app/locales/sv.json]: It's present in the other languages, so I assume it should be there.

**Comment by mihai-sysbio** [comment]
> `app/domain/urls.ts` must have se added

There are discussion regarding switching to `sv-se` or the like but `se` is great for now.

**Comment by talavis** [comment]
The zipcodes etc were added as se, so I got the impression it was already decided.

File: app/locales/en.json (hunk 1 above)
Comment (MalinAhlberg) [reply app/locales/en.json]: Oops, sorry. Was to tired and added it to the wrong file. It's now in the `se.json`.

**Review by pontus — APPROVED** [review]
LGTM

**Pull request merged**